bin/storm.py

```diff
@@ -154,6 +157,39 @@ def confvalue(name, extrapaths, daemon=True):
             return " ".join(tokens[1:])
     return ""
 
+def resolve_dependencies(packages):
```

Comment:
nit: It would be nice to have a short circuit here so if there are no dependencies to resolve we don't have to call into another java application.